bundles/cif-connector-graphql/src/test/java/com/adobe/cq/commerce/graphql/resource/GraphqlResourceProviderTest.java

```diff
@@ -122,11 +124,22 @@ public void setUp() throws Exception {
         when(jcrResourceProvider.getResource(resolveContext, CATALOG_ROOT_PATH, null, null)).thenReturn(rootResource);
         when(rootResource.getPath()).thenReturn(CATALOG_ROOT_PATH);
         when(rootResource.getValueMap()).thenReturn(rootValueMap);
+        when(rootResource.getResourceResolver()).thenReturn(resourceResolver);
 
         scheduler = mock(Scheduler.class);
         ScheduleOptions opts = mock(ScheduleOptions.class);
         when(scheduler.schedule(any(), any())).thenReturn(Boolean.FALSE);
         when(scheduler.NOW(Mockito.anyInt(), Mockito.anyLong())).thenReturn(opts);
+
+        provider = new GraphqlResourceProvider<>(CATALOG_ROOT_PATH, dataService, scheduler);
+        when(resourceResolver.getResource(any())).then(invocationOnMock -> {
+            String path = (String) invocationOnMock.getArguments()[0];
+            if (path.endsWith("/.")) {
```

Comment:
Thx for moving this to the test code. Just to clarify, why do we need that? It seems looks odd to me that a path would end with `/.`. Thx.

Reply:
DeepReadValueMapDecorator tries to resolve a path ending in "/."